compiler/rustc_interface/src/interface.rs

```diff
@@ -275,7 +275,9 @@ pub(crate) fn parse_check_cfg(dcx: DiagCtxtHandle<'_>, specs: Vec<String>) -> Ch
                     .expecteds
                     .entry(name.name)
                     .and_modify(|v| match v {
-                        ExpectedValues::Some(v) if !values_any_specified => {
+                        ExpectedValues::Some(v) if !values_any_specified =>
+                        {
```

Comment:
```suggestion
                        ExpectedValues::Some(v) if !values_any_specified => {
```
why does this formatting change happen?